infra/gravity/cluster_install.go

```diff
@@ -205,7 +205,7 @@ func (c *TestContext) uploadInstaller(master Gravity, nodes []Gravity, installer
 		return trace.Wrap(err)
 	}
 
-	err = c.Status(nodes)
+	_, err = c.Status(nodes)
```

Comment:
This previously looped, waiting for `!= degraded`.  Now it will check only once.  We can't use to `c.WaitForActiveStatus(nodes)` because I don't believe that `gravity status` returns active before the install.  Perhaps we need another wait function?

Reply:
This is responsible for the upgrade failure seen [here](https://console.cloud.google.com/logs/viewer?authuser=0&expandAll=false&project=kubeadm-167321&minLogLevel=0&timestamp=2020-05-28T23:07:45.203000000Z&customFacets=&limitCustomFacetWidth=true&advancedFilter=resource.type%3D%22project%22%0Alabels.__uuid__%3D%22dbc09cee-cb33-4304-993e-7db324cec2a0%22%0Alabels.__suite__%3D%22193da8e2-b2cc-4439-8855-b7f80c6ae3a2%22&dateRangeStart=2020-05-28T22:07:57.117Z&dateRangeEnd=2020-05-28T23:07:57.117Z&interval=PT1H&scrollTimestamp=2020-05-28T22:41:17.837368627Z).  Fixed in e4c7185